src/osx/carbon/cursor.cpp

```diff
@@ -278,7 +278,16 @@ wxCursor::wxCursor(const wxString& cursor_file, wxBitmapType flags, int hotSpotX
     if ( flags == wxBITMAP_TYPE_MACCURSOR_RESOURCE )
     {
 #if wxOSX_USE_COCOA
-        wxFAIL_MSG( wxT("Not implemented") );
+        wxString fileName = wxStandardPaths::Get().GetResourcesDir() + "/" + cursor_file + ".png";
+        wxImage image( fileName, wxBITMAP_TYPE_PNG );
+        if( image.IsOk() )
+        {
+            m_refData->DecRef() ;
+            m_refData = NULL ;
```

Comment:
for PNG we need to set the hotspot explicitly in the image as in the `else` clause, also bracket with `wxUSE_IMAGE`

Reply:
@csomor ,
I will verify it tonight, but I think it will be set in the "InitFromImage()".

Reply:
No wxImage code can only set it for a format that itself does know about the hotspot location, eg CUR, but not for PNG which does not carry this information

Reply:
@csomor,
Got it.
Do yo think it is a good idea to add a fallback to CUR file if PNG is not found?